src/displayBackend/drm/Display.cpp

```diff
@@ -244,6 +250,96 @@ FrameBufferPtr Display::createFrameBuffer(DisplayBufferPtr displayBuffer,
  * Private
  ******************************************************************************/
 
+bool Display::isDrmDevice(udev_device *device)
+{
+	const char *filename = udev_device_get_devnode(device);
+	const char *sysnum = udev_device_get_sysnum(device);
```

Comment:
It seems to me if sysnum is NULL here there is a little sense in opening file, retrieving resources, etc.